### ISSUE TITLE
Remove develop branch

### ISSUE THREAD

**Issue by kakawait** [kakawait/hugo-tranquilpeak-theme#484]
Lot of work mess for nothing

- [ ] #481

**Comment by stiliajohny** [comment]
hi, @kakawait have you consider using GitHub actions to make life easier ???

**Comment by kakawait** [comment]
Yes but to be honest lake of time to invest for the moment. I'd like to concentrate on things that I've logged since few years.

**Comment by stiliajohny** [comment]
I would be happy to put something together on Github Actions ;)

**Comment by kakawait** [comment]
> I would be happy to put something together on Github Actions ;)

My dream will be per PR as something to run `hugo` (at least to check no basic errors), eslint and then deploy the result somewhere (maybe netlify, see #488) as _Preview app_ to get a visual render before merging to master.

**Comment by kakawait** [comment]
But currently the main problem of develop branch is time between feature/PR is merged and get "publicly" accessible. I don't think is worth keeping/buffering everything in develop branch.

Most people except that after proposing PR (and accepted/merged) to get result asap and not wait for the maintainer (me) to release the new version

**Comment by stiliajohny** [comment]
agree, 

I am truly an advocate of google's approach of emerging in master if works and if broke fix forwards.

**Comment by stiliajohny** [comment]
I am gonna fork and get some GitHub actions that do the tests... 
very theoretically,

I would have got the following checks. 

Markdown Link checker ( for validation the URL is valid ) 
ESlint 
Hugo check 
Update PR with a bot comment 

this kind of thing
